client/scraper/drive_metadata.py

```diff
@@ -0,0 +1,34 @@
+from client import config
+from metadata import Metadata
+import json
+import requests
+import re
+
+request_url = "https://www.googleapis.com/drive/v2/files/"
+
+
+class DriveMetadata(Metadata):
+    prop_map = {}
+
+    def get_json_response(self, document_id):
```

Comment:
get_json_response really shouldn't exist in Metadata... if anything we should make it a method of the Response class.